app/templates/components/ui-table/cell/events/view/speakers/speaker-mobile.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if (not-eq record.mobile 'null')}}
```

Comment:
No need, let null be there. Empty space looks odder.

Reply:
Can we use something like `Not Provided`?

Reply:
okay, go ahead.